### PR TITLE
Ihsanmertatalay

### PR DESCRIPTION
**Pull request by ihsanmertatalay**
I added my project but my code in the view.py it is ok in my local but may have some mistakes in there
and   added index.html in template

### CODE REVIEW

File: practice-app/api/views.py [hunk 1]
```diff
@@ -9,6 +10,7 @@
 from api.places.places import places_api
 from api.random_article.main import ra_api
 from api.eq_post.main import eq_post_api
+from havalı.hava.hava import hava_api
```
Comment (kerimoglutolga) [inline practice-app/api/views.py:13]: Shouldn't this import be 'from api.hava.hava import hava_api' ?

File: practice-app/api/urls.py [hunk 2]
```diff
@@ -9,4 +9,6 @@
     path('eq_post/', view=views.eq_post, name='eq_post'),
     path('covid19/', view=views.covid19, name='covid19'),
     path( 'covid19/<str:countrycode>', views.covid_country, name="covid_country"),
+     path('', views.hava_api),
```
Comment (kerimoglutolga) [inline practice-app/api/urls.py:12]: You should add an extension to the url for your particular api. Ex: path('hava/', view=views.hava, name="hava").

File: practice-app/api/views.py (hunk 1 above)
Comment (ihsanmertatalay) [reply practice-app/api/views.py]: ok I change it

File: practice-app/api/urls.py (hunk 2 above)
Comment (ihsanmertatalay) [reply practice-app/api/urls.py]: ok I changed it

**Pull request merged**